end_area: 267.46
end_frontier: 0
end_time: 3.6e+07
